os/android/iahwc2.cpp

```diff
@@ -1083,6 +1083,14 @@ int IAHWC2::HookDevOpen(const struct hw_module_t *module, const char *name,
   return 0;
 }
 
+hwcomposer::NativeDisplay *IAHWC2::GetDisplay() {
+  return extended_displays_.at(0).GetDisplay();
```

Comment:
Primary display is not part of this list. Main display should primary_display_. You want to have two API, one return Primary display, and another one returning vector of extended displays, which u can use than to print the supported display configs.